templates/private/approve_names.html

```diff
@@ -0,0 +1,140 @@
+{% extends "private/admin_base.html" %}
+{% block title %}Name Approval{% endblock %}
+
+{% block content %}
+{# LOGO #}
+<div class="container" style="margin-top: 50px">
+
+    {# Display logo #}
+    <div style="margin-bottom: 50px">
+        <img
+                src="{{ url_for('static', filename=resources.LOGO) }}"
+                class="center-block img-responsive mx-auto d-block"
+                alt="Logo"
+                width="100"
+                height="100"
+        >
+    </div>
+
+</div>
+
+{# Key creation #}
+<div class="container">
+    <div class="text-center">
+        <h3>Name Approval</h3>
+    </div>
+
+    <div class="col-lg-3">
+    </div>
+    <div class="col-lg-6">
+
+    {# If the error value is set, display the error in red text #}
+    {% if error %}
+        <div class="alert alert-danger">
+            {{ error }}
+        </div>
+    {% endif %}
+
+    {# If the message value is set, display the message in green text #}
+    {% if message %}
+        <div class="alert alert-success">
+            {{ message }}
+        </div>
+    {% endif %}
+
+    {% if names %}
+    <h3>Character Names</h3>
+    <table class="table table-striped">
```

Comment:
What do you think about including some default text like "No pending approvals" when there are neither pending character name nor pet name approvals?
This would make it a little bit more obvious why the page is empty.

Though I am not a member of this project, so feel free to just ignore that suggestion.